### PR TITLE
EWPP-31: Make Departments a shared field.

### PR DESCRIPTION
**Pull request by julien-**

### CODE REVIEW

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]
The field template file has not been updated (field--node--oe-project-departments--oe-project.html.twig)

**Comment by julien-** [comment]
> The field template file has not been updated (field--node--oe-project-departments--oe-project.html.twig)

File renamed to `field--node--oe-departments--oe-project.html.twig`

**Review by imanoleguskiza — DISMISSED** [review]

File: modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml [hunk 1]
```diff
@@ -119,6 +119,14 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  oe_departments:
+    type: skos_concept_entity_reference_label
+    weight: 6
+    region: content
+    label: above
+    settings:
+      link: true
```
Comment (nagyad) [inline modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml]: I think this should not be a clickable link, is it?

**Review by nagyad — CHANGES_REQUESTED** [review]

**Review by nagyad — DISMISSED** [review]

File: modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml (hunk 1 above)
Comment (julien-) [reply modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml]: Fixed

**Review by nagyad — APPROVED** [review]

**Review by ademarco — APPROVED** [review]

**Pull request merged**